functions/ai/resume_parser.py

```
@@ -0,0 +1,87 @@
+"""
+AI service for parsing resumes.
+"""
+import io
+import json
+from typing import Dict, Any
+
+from genkit import ai, flow
```

Ruff (0.12.2)

`genkit.ai` imported but unused

Remove unused import: `genkit.ai`

(F401)